src/lua/crypto.lua

```diff
@@ -28,10 +28,11 @@ ffi.cdef[[
     HMAC_CTX *crypto_HMAC_CTX_new(void);
```

Comment:
Nit: Maybe change `HMAC_CTX` here to `crypto_HMAC_CTX` for the parity with the C code?

Reply:
Okay.